src/main/java/com/example/nlern/controllers/subloadController.java

```diff
@@ -1,23 +1,132 @@
 package com.example.nlern.controllers;
 
-import com.example.nlern.models.User;
+import com.example.nlern.models.FullTest;
+import com.example.nlern.models.Question;
+import com.example.nlern.models.Test;
+
 import com.example.nlern.repos.GroupRepository;
+import com.example.nlern.repos.QuestionRepository;
+import com.example.nlern.repos.TestRepository;
+import com.example.nlern.repos.UserRepository;
+
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.http.ResponseEntity;
+import org.springframework.ui.Model;
+
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.DeleteMapping;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.util.List;
 
 @RestController
+@CrossOrigin
 @RequestMapping("/subload")
-public class subloadController {
+public class SubloadController {
```

Comment:
Change file name so it's the same with the class name

Reply:
Done

Reply:
You should also change this controller so it uses only the logic from respected services and does not have a direct access  to repositories